### ISSUE TITLE
0.12: only primary monitor displays background

### ISSUE THREAD

**Issue by Soukyuu** [sddm/sddm#502]
Probably related to #496, but unlike it I don't have any login issues. I'm running arch x64 with two monitors and legacy 340xx nvidia blob. Ever since upgrading to 0.12, only the primary monitor displays the wallpaper on the login screen. Downgrading to 0.11 fixes the problem.

**Comment by vdemin** [comment]
@Soukyuu Are your monitors the same size? I have two different-sized monitors and see only part of image on second screen. Seems the size of this part is the difference between the sizes of monitors:

![sddm-dual-monitors](https://cloud.githubusercontent.com/assets/1164905/10160868/f884519a-66a8-11e5-9429-ea72cbc76b3a.jpg)

```
% xrandr -q            
Screen 0: minimum 8 x 8, current 4480 x 1200, maximum 8192 x 8192
DVI-I-0 disconnected (normal left inverted right x axis y axis)
VGA-0 disconnected (normal left inverted right x axis y axis)
DVI-I-1 connected primary 1920x1200+0+0 (normal left inverted right x axis y axis) 518mm x 324mm
   1920x1200     59.95*+
   1920x1080     60.00  
   1680x1050     59.95  
   1600x1200     60.00  
   1280x1024     60.02  
   1280x960      60.00  
   1024x768      60.00  
   800x600       60.32  
   640x480       59.94  
HDMI-0 connected 2560x1080+1920+120 (normal left inverted right x axis y axis) 677mm x 290mm
   2560x1080     60.00*+
   1920x1080     60.00    59.94    50.00    29.97    60.05    60.00    50.04  
   1680x1050     59.95  
   1600x900      60.00  
   1280x1024     75.02    60.02  
   1280x720      60.00    59.94    50.00  
   1152x864      75.00  
   1024x768      75.03    60.00  
   800x600       75.00    60.32  
   720x576       50.00  
   720x480       59.94  
   640x480       75.00    59.94    59.93
```
I also have arch x64, legacy 340xx nvidia blob and no login issues.

**Comment by Soukyuu** [comment]
No,they're not:
```
Screen 0: minimum 8 x 8, current 3200 x 1084, maximum 8192 x 8192
DVI-I-0 disconnected (normal left inverted right x axis y axis)
DVI-I-1 disconnected (normal left inverted right x axis y axis)
DVI-I-2 connected primary 1920x1080+0+0 (normal left inverted right x axis y axis) 531mm x 299mm
   1920x1080     59.93 +  60.00* 
   1680x1050     59.95  
   1280x1024     75.02    60.02  
   1280x960      60.00  
   1152x864      75.00  
   1024x768      75.03    60.00  
   800x600       75.00    60.32    56.25  
   640x480       75.00    59.94  
DVI-I-3 connected 1280x1024+1920+60 (normal left inverted right x axis y axis) 330mm x 270mm
   1280x1024     60.02*+
   1024x768      60.00  
   800x600       60.32  
   640x480       59.95    59.94 
```
Mine are more similarly sized, though, but I don't see any part of the image on the right screen.

**Comment by plfiorini** [comment]
@Soukyuu what theme are you using?

**Comment by Soukyuu** [comment]
the default Plasma 5 one, Breeze 0.1 by David Edmundson.

**Comment by membrive** [comment]
I have the same issue, with the same theme and configuration.

**Comment by plfiorini** [comment]
Please try with: https://github.com/sddm/sddm/pull/511

**Comment by plfiorini** [comment]
Should be fixed with #511, please reopen if it's not.
